mathesar/migrations/0007_auto_20221115_0627.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.1.14 on 2022-11-15 06:27
+
+from django.db import migrations, models
+import mathesar.models.query
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mathesar', '0006_auto_20221110_1152'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='uiquery',
+            name='description',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='uiquery',
+            name='display_options',
+            field=models.JSONField(blank=True, null=True, validators=[mathesar.models.query._get_validator_for_dict]),
+        ),
+        migrations.AlterField(
+            model_name='uiquery',
+            name='initial_columns',
+            field=models.JSONField(validators=[mathesar.models.query._get_validator_for_list_of_dicts, mathesar.models.query._get_validator_for_initial_columns]),
+        ),
+        migrations.AlterField(
+            model_name='uiquery',
+            name='transformations',
+            field=models.JSONField(blank=True, null=True, validators=[mathesar.models.query._get_validator_for_list_of_dicts, mathesar.models.query._get_validator_for_transformations]),
+        ),
```

Comment:
I don't know why this is getting added to the migration. It is similar to 0005 and 0006. and both 0005 and 0006 are actually the same, everytime I run `makemigrations` a new migration file is created with these fields.

Reply:
For now, I think we should merge this with the extra migration steps.

Please make a bug ticket to figure out what's going on here.